.github/workflows/integration-tests-non-root.yaml

```
@@ -0,0 +1,69 @@
+name: integration-tests-non-root
+run-name: Integration tests (non-root) - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    paths:
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+
+jobs:
+  test:
+    name: Integration tests (non-root)
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      id-token: write
+
+    container: 
+      image: public.ecr.aws/gravitational/teleport-buildbox:teleport11
```

Should probably use `teleport12` buildbox as this is master.
```suggestion
      image: public.ecr.aws/gravitational/teleport-buildbox:teleport12
```